packages/storage-layer-torus/src/TorusStorageLayer.ts

```diff
@@ -127,10 +138,19 @@ class TorusStorageLayer implements IStorageLayer {
     let sig: string;
     let pubX: string;
     let pubY: string;
+    let namespace = "tkey";
     const setTKeyStore = {
       data: message,
       timestamp: new BN(~~((this.serverTimeOffset + Date.now()) / 1000)).toString(16),
     };
+
+    // Overwrite bulk_set to allow deleting nonce v2 together with creating tKey.
+    // This is a workaround, a better solution is allow upstream API to set tableName/namespace of metadata params
```

Comment:
i believe lets do this as well

Reply:
needa fix the tests too @phuctm97

Reply:
fixed this

Reply:
metadata-js and direct-web-sdk need to be merged first for tests to pass